derive/src/client_def.rs

```diff
@@ -231,18 +256,18 @@ impl State {
 		});
 
 		quote! {
-			fn verify_client_consensus_state<Ctx: ReaderContext>(
+			fn verify_client_consensus_state<Ctx: #crate_::core::ics26_routing::context::ReaderContext>(
```

Comment:
This isn't going to work for downstream crates, it should be 

`::ibc::core::ics26_routing::context::ReaderContext`

Reply:
You're right, thanks

Reply:
`crate_` here is an identifier of the `ibc` crate defined by the user so that it will work (or `crate` keyword in case of using the macro in the `ibc` crate in mocks). The only difference is that there is no prepending `::`, I'll add it

Reply:
ok didn't know that, great thinking